frontend/src/lib/components/country_selector.svelte

```diff
@@ -5,6 +5,7 @@
   import { get } from "svelte/store"
 
   export let countries: main_Country[]
+  export let handleDropdownClick: Function | undefined = undefined
```

Comment:
This seems redundant - Does it have to default to `undefined`?

```suggestion
  export let handleDropdownClick: Function
```